Source/Scene/Cesium3DTileFeature.js

```diff
@@ -69,10 +82,21 @@ define([
          */
         show : {
             get : function() {
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
+                    return label.show;
+                }
                 return this._batchTable.getShow(this._batchId);
             },
             set : function(value) {
-                this._batchTable.setShow(this._batchId, value);
+                if (defined(this._labelCollection)) {
```

Comment:
Style-wise throughout when members like `this._labelCollection` and `this._batchId` are used more than once in a function assign them to a local.